scripts/partition_manager.py

```diff
@@ -416,6 +416,29 @@ def first_partition_has_been_aligned(first, solution):
 
 
 def _set_addresses_and_align(reqs, sub_partitions, solution, size, start, dynamic_partitions, dp):
+    # Resolve 'align_next'
+    solution_non_empty = list(filter(lambda part: "EMPTY_" not in part, solution))
+    for i in range(len(solution_non_empty) - 1):
```

Comment:
I'm probably missing something here, but are we intentionally leaving out the last element in the list?